front_end/src/components/payment/PaymentPage.css

```diff
@@ -0,0 +1,90 @@
+/* Payment Page */
+.payment-row {
+  display: -ms-flexbox; /* IE10 */
+  display: flex;
+  -ms-flex-wrap: wrap; /* IE10 */
+  flex-wrap: wrap;
+  margin: 0 -16px;
+  color: aqua;
+}
+
+.payment-col-25 {
```

Comment:
I don't know if I missed it, but these 'payment-col' classes aren't being used in BookingInfo, PaymentPage or Payment.

Reply:
very possible, I just copied over Yuta's file. didn't look into the css much. The styling of the payment page will be a separate story, but I will take out the unused css classes for now 👍